### DISCUSSION TITLE
Problem with Raspberry Pi 3

### DISCUSSION THREAD

**Discussion by kalto** [mKeRix/room-assistant#340] [Q&A]
I'm running Room-Assistsant with two Raspbarry Pi, a 3 and a 4.

Everything works flawlessly on the 4 but it stops working on the 3 after a couple of minutes and log this in dmesg:

[   69.235259] debugfs: File 'le_min_key_size' in directory 'hci0' already present!
[   69.235279] debugfs: File 'le_max_key_size' in directory 'hci0' already present!

The only to get it to work again is to reboot the Pi. 

Any idea?

Thanks!

**Comment by mKeRix** [comment]
Could you share some more details for your installation (operating system, room-assistant version, enabled integrations)? Is this happening for the Home Assistant add-on version of room-assistant by any chance?
